bundles/AdminBundle/Resources/public/js/pimcore/document/tags/input.js

```diff
@@ -90,13 +94,14 @@ pimcore.document.tags.input = Class.create(pimcore.document.tag, {
             this.element.dom.setAttribute('data-placeholder', options["placeholder"]);
         }
 
+        // @TODO validator is based on \Zend\Json\Expr and does not work with Twig templates, to be removed in v7.0
         if(options["validator"]) {
             this.element.isValid = options["validator"];
             this.validateElement();
         }
     },
 
-    checkValue: function () {
+    checkValue: function (mark) {
```

Comment:
The `mark` argument seems to be useless, as `validateValue()` is called only when `this.required == true` and in this case `mark` is always `true`, so I'd say we can just call `this.validateValue(value, this.element, this, true);` instead. 

Please also change that for the other editables.

Reply:
We still need mark argument as on initial load checkValue() is called which also checks the required editables but it should not mark element as required(red highlight class).

Reply:
ok, thanks for clarification